<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ArduCAM</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBoxResol">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pix :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>64</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Baudrate :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxBaud">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>104</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPort">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>254</x>
      <y>20</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbUp">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>155</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Up&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="imagenes/images.qrc">
      <normaloff>:/new/imagenes/au.png</normaloff>:/new/imagenes/au.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRight">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>190</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Right&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="imagenes/images.qrc">
      <normaloff>:/new/imagenes/ar.png</normaloff>:/new/imagenes/ar.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbZero">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Home&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="imagenes/images.qrc">
      <normaloff>:/new/imagenes/zero.png</normaloff>:/new/imagenes/zero.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLeft">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>190</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Left&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="imagenes/images.qrc">
      <normaloff>:/new/imagenes/al.png</normaloff>:/new/imagenes/al.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDown">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>224</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Down&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="imagenes/images.qrc">
      <normaloff>:/new/imagenes/ad.png</normaloff>:/new/imagenes/ad.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Servos :</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>592</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionSpray"/>
   <addaction name="actioncolorIdentification"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="imagenes/images.qrc">
     <normaloff>:/new/imagenes/connect.png</normaloff>:/new/imagenes/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionSpray">
   <property name="icon">
    <iconset resource="imagenes/images.qrc">
     <normaloff>:/new/imagenes/spray.png</normaloff>:/new/imagenes/spray.png</iconset>
   </property>
   <property name="text">
    <string>Spray</string>
   </property>
   <property name="toolTip">
    <string>Spray</string>
   </property>
  </action>
  <action name="actioncolorIdentification">
   <property name="icon">
    <iconset resource="imagenes/images.qrc">
     <normaloff>:/new/imagenes/imageSegmentation.png</normaloff>:/new/imagenes/imageSegmentation.png</iconset>
   </property>
   <property name="text">
    <string>colorIdentification</string>
   </property>
   <property name="toolTip">
    <string>Color Identification</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="imagenes/images.qrc"/>
 </resources>
 <connections/>
</ui>
